go/cache/theine/singleflight_test.go

```diff
@@ -54,8 +54,8 @@ func TestDoErr(t *testing.T) {
 		return "", someErr
 	})
 
-	assert.ErrorIs(t, err, someErr, "incorrect Do error")
-	assert.Empty(t, v, "unexpected non-empty value")
+	assert.Equal(t, someErr, err)
```

Comment:
why did we change from `ErrorIs` to `Equal` here?

Reply:
this may have been due to me using the test-rewriter and then resolving merge conflicts when rebasing. I'll revert this